db/seeds/base/add_applications.rb

```diff
@@ -1,27 +1,62 @@
-all_lead_providers = LeadProvider.all
+# frozen_string_literal: true
 
-# users with one application each
-FactoryBot.create_list(
-  :application,
-  20,
-  :with_random_user,
-  :with_random_work_setting,
-  ecf_id: SecureRandom.uuid,
-  lead_provider: all_lead_providers.last,
-  course: Course.all.sample,
-  lead_provider_approval_status: "accepted",
-  participant_outcome_state: "passed",
-)
+LeadProvider.find_each do |lead_provider|
+  quantity = { "review" => 4, "development" => 1 }.fetch(Rails.env, 0)
 
-# a user with 4 applications
-FactoryBot.create_list(
-  :application,
-  4,
-  :with_random_lead_provider_approval_status,
-  :with_random_participant_outcome_state,
-  :with_random_work_setting,
-  user: FactoryBot.create(:user, :with_random_name),
-  ecf_id: SecureRandom.uuid,
-  lead_provider: all_lead_providers.sample,
-  course: Course.all.sample,
-)
+  quantity.times do
+    # users with one application each
+    FactoryBot.create_list(
+      :application,
+      10,
+      :with_random_user,
+      :with_random_work_setting,
+      ecf_id: SecureRandom.uuid,
+      lead_provider:,
+      course: Course.all.sample,
+      lead_provider_approval_status: "pending",
+      cohort: Cohort.all.sample,
+    )
+
+    # users with 4 applications each
+    FactoryBot.create_list(
+      :application,
+      4,
+      :with_random_lead_provider_approval_status,
+      :with_random_participant_outcome_state,
+      :with_random_work_setting,
+      user: FactoryBot.create(:user, :with_random_name),
+      ecf_id: SecureRandom.uuid,
+      lead_provider:,
+      course: Course.all.sample,
+      cohort: Cohort.all.sample,
+    )
+
+    # users with one accepted application each
+    FactoryBot.create_list(
+      :application,
+      4,
+      :with_random_user,
+      :with_random_work_setting,
+      ecf_id: SecureRandom.uuid,
+      lead_provider:,
+      course: Course.all.sample,
+      lead_provider_approval_status: "accepted",
+      participant_outcome_state: "passed",
+      cohort: Cohort.all.sample,
+    )
+
+    # users with one rejected application each
+    FactoryBot.create_list(
+      :application,
+      4,
+      :with_random_user,
+      :with_random_work_setting,
+      ecf_id: SecureRandom.uuid,
+      lead_provider:,
+      course: Course.all.sample,
+      lead_provider_approval_status: "rejected",
+      participant_outcome_state: "passed",
```

Comment:
Curious about this value, is this related to our outcomes from ECF? If it is maybe we set it to false on rejected?